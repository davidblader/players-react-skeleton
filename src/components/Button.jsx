import React from 'react';
import PropTypes from 'prop-types';

const Button = props => (<button {...props} className="animal-crossing-btn">{props.children}</button>);

Button.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Button;
