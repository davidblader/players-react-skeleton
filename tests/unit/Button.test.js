import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../src/components/Button';

test('Button shallow renders', () => {
  const component = shallow(<Button>Test</Button>);
  expect(component).toMatchSnapshot();
});
