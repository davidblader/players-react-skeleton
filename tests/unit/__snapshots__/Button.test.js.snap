// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button shallow renders 1`] = `
<button
  className="animal-crossing-btn"
>
  Test
</button>
`;
